# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/imgLoader.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 image/imgLoader.cpp
--- a/image/imgLoader.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/image/imgLoader.cpp	Sun May 26 14:30:14 2019 +0000
@@ -15,6 +15,7 @@
 #include "mozilla/Move.h"
 #include "mozilla/NullPrincipal.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/ChaosMode.h"
 #include "mozilla/LoadInfo.h"
 #include "mozilla/StaticPrefs.h"
@@ -55,7 +56,6 @@
 #include "nsIMemoryReporter.h"
 #include "DecoderFactory.h"
 #include "Image.h"
-#include "gfxPrefs.h"
 #include "prtime.h"
 #include "ReferrerInfo.h"
 
@@ -1251,8 +1251,8 @@
 }
 
 void imgLoader::GlobalInit() {
-  sCacheTimeWeight = gfxPrefs::ImageCacheTimeWeight() / 1000.0;
-  int32_t cachesize = gfxPrefs::ImageCacheSize();
+  sCacheTimeWeight = StaticPrefs::ImageCacheTimeWeight() / 1000.0;
+  int32_t cachesize = StaticPrefs::ImageCacheSize();
   sCacheMaxSize = cachesize > 0 ? cachesize : 0;
 
   sMemReporter = new imgMemoryReporter();